docs/serverless/advanced-entity-analytics/ers-req.mdx

```diff
@@ -41,8 +41,6 @@ To turn on the risk scoring engine, you need either the appropriate <DocLink slu
 
 ## Asset criticality
```

Comment:
I'm confused because in the preview this says Entity Risk Scoring but here it says Asset Criticality. Not a problem I guess since the preview looks good? Maybe I'm confused

Reply:
Sorry, should have linked to the exact section – https://security-docs_bk_5991.docs-preview.app.elstc.co/guide/en/security/master/ers-requirements.html#_asset_criticality. I'll edit the link in the description.